docs/pages/pmd/devdocs/pmd_devdocs_metrics_howto.md

```diff
@@ -0,0 +1,16 @@
+---
+title:  Using code metrics in custom rules
+tags: [customizing]
+summary: "PMD was recently enhanced with the ability to compute code metrics on Java code (the so-called Metrics 
+Framework). This framework provides developers with a straightforward interface to use code metrics in their rules, and 
+to extend the framework with their own custom metrics."
+last_updated: July 20, 2017
+sidebar: pmd_sidebar
+permalink: pmd_devdocs_metrics_howto.html
```

Comment:
I've meanwhile changed my naming convention: since we have "pmd" and "devdocs" already in the path as folder, I decided to do not repeat these in the filename. So, please rename this file to just `metrics_howto.md`.
Important: Keep the "permalink" as it is - this will be the final name of the html file, that is generated. Since this jekyll theme generates all pages flat in the root directory, it's easier to see then where this page belongs to (and avoid any clashes).